examples/with-apollo/lib/initApollo.js

```diff
@@ -0,0 +1,37 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
```

Comment:
I still think would be nice to share the files between both examples instead of duplicating all code...

E.g I think that this is the same code as in the other `initApollo.js` file so I'm not reviewing it, but it might in fact contain issues etc.

DRY you know ;)